Dart/src/com/jetbrains/lang/dart/validation/fixes/DartServerFixIntention.java

```diff
@@ -180,9 +184,23 @@ public boolean startInWriteAction() {
   @Override
   public void invoke(@NotNull final Project project, final Editor editor, final PsiFile file) throws IncorrectOperationException {
     final SourceFileEdit fileEdit = myChange.getEdits().get(0);
+    final String filePath = fileEdit.getFile();
```

Comment:
Yes, probably.
We still provide full names if the client wants to show it.
https://codereview.chromium.org/1147733002